workshop/01-foldDyn.md

```diff
@@ -171,11 +219,11 @@ We wire this up in a recursive do block so that we can feed the dynamic that we
 
 The the variables that are defined in the rec can have cycles, but are still defined at the same level as the display. It's good to keep the rec blocks as small as is possible so that causality loops (FRP infinite loops) are easier to spot and fix.
 
-Using rec or mdo forces that context to have MonadFix (the abstraction used to tie the knot in the cycle). Doco about the syntax can be found [on the haskell wiki](https://wiki.haskell.org/MonadFix).
+Using rec or mdo forces that context to have `MonadFix` (the abstraction used to tie the knot in the cycle). Documenation about the syntax can be found [on the haskell wiki](https://wiki.haskell.org/MonadFix).
```

Comment:
`rec` and `mdo` are keywords too, so please code-quote them.

Reply:
Also typo "documentation".